### PR TITLE
RavenDB-14891 Expose IdentityPartsSeparator in Client Configuration i…

### PR DESCRIPTION
**Pull request by Danielle9897**
…n Studio

### CODE REVIEW

File: src/Raven.Studio/typescript/models/database/settings/clientConfigurationModel.ts [hunk 1]
```diff
@@ -24,6 +29,11 @@ class clientConfigurationModel {
     constructor(dto: Raven.Client.Documents.Operations.Configuration.ClientConfiguration) {
 
         if (dto) {
+            if(dto.IdentityPartsSeparator) {
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/settings/clientConfigurationModel.ts]: space after if

File: src/Raven.Studio/wwwroot/App/views/manage/clientConfiguration.html [hunk 2]
```diff
@@ -8,30 +8,49 @@
...
                     </div>
                     <div class="panel padding">
                         <div class="row">
                             <div class="col-sm-6">
                                 <div class="checkbox">
-                                    <input id="useSessionContextToggle" type="checkbox" 
-                                           data-bind="checked: model.isDefined, checkedValue: 'useSessionContextForLoadBehavior', disable: _.includes(model.isDefined() , 'readBalanceBehavior')">
-                                    <label for="useSessionContextToggle">
+                                    <input id="checkboxUseSessionContext" type="checkbox"
+                                           data-bind="checked: model.isDefined, checkedValue: 'useSessionContextForLoadBehavior', disable: _.includes(model.isDefined() , 'readBalanceBehavior')" />
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/manage/clientConfiguration.html]: Didn't we agree that we don't want to disable this option? But when we choose this option we uncheck second toggle.

**Review by ml054 — CHANGES_REQUESTED** [review]

File: src/Raven.Studio/typescript/models/database/settings/clientConfigurationModel.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/settings/clientConfigurationModel.ts]: done

File: src/Raven.Studio/wwwroot/App/views/manage/clientConfiguration.html (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/manage/clientConfiguration.html]: fixed

**Review by ml054 — APPROVED** [review]

**Pull request merged**